src/main/resources/db/changelog/changes/08-create-shopping-carts-table.yml

```diff
@@ -9,14 +9,12 @@ databaseChangeLog:
               - column:
                   name: id
                   type: bigint
-                  autoIncrement: true
                   constraints:
                     nullable: false
                     primaryKey: true
               - column:
-                  name: user_id
-                  type: bigint
+                  name: is_deleted
```

Comment:
Why you delete user_id column?

Reply:
i use mapsId and user_id is the same as shopping_cart_id; and it works without additional column 'user_id'